operator/pkg/reconciler/reconciler_factory.go

```diff
@@ -83,6 +84,11 @@ func SetupReconcilerWithManager(mgr ctrl.Manager, gvk schema.GroupVersionKind, c
 		client.AppendPostRenderers(func(rm meta.RESTMapper, kubeClient kube.Interface, obj ctrlClient.Object) postrender.PostRenderer {
 			return overlays.NewOverlayPostRenderer(obj, obj.GetNamespace())
 		}),
+		client.AppendPostRenderers(func(rm meta.RESTMapper, kubeClient kube.Interface, obj ctrlClient.Object) postrender.PostRenderer {
+			return commonLabels.NewLabelPostRenderer(kubeClient, map[string]string{
+				"app.stackrox.io/managed-by": "operator",
```

Comment:
I didn't use that because it's already used by the underlying helm libraries to set `app.kubernets.io/managed-by: Helm`. I didn't want to risk strange side effects by overwriting this label. And I couldn't reuse it because we don't want to watch all secrets created by any helm installation on a cluster.